### ISSUE TITLE
Description?

### ISSUE THREAD

**Issue by voltrevo** [opentok/annotation-acc-pack#99]
Hey guys. I think this project is missing a basic description in README.md. I've been trying to get this working for some time and I'm still kinda lost about what it's supposed to do and look like when it's working. It seems to be assumed knowledge what is meant by 'annotation'. I can guess that means drawing something (what?) on top of a stream (what kind?) that can be seen by other participants, but how this is achieved is a mystery.

When I tried to use it, I got this:

![screen shot 2016-10-26 at 4 16 56 pm](https://cloud.githubusercontent.com/assets/9291586/19714405/9236a290-9b99-11e6-924a-625315225453.png)

The exceptions indicate something is still wrong in my set-up. But, if it was working correctly, would the background be a solid color like this? Or should a stream be playing there for me to draw on top? I'd absolutely love to see a screenshot in the readme. You can include it in markdown like this:

```md
![Alt text](path to image image - this can be a relative path in the repo itself)
```

**Comment by adrice727** [comment]
@voltrevo This was fixed in commit [55225a](https://github.com/opentok/annotation-acc-pack/commit/552225ae232556a685b8486afa3094ff7261a5f1).  `npm update` should take care of it.

**Comment by voltrevo** [comment]
@adrice727 I'm confused, this issue is about `README.md`, but there are only code changes in https://github.com/opentok/annotation-acc-pack/commit/552225ae232556a685b8486afa3094ff7261a5f1.

**Comment by adrice727** [comment]
@voltrevo Yeah, sorry.  The changes in `55225a` fix the issue in the screenshot.  Let's leave this open until we update the README.
